servicecontrol/servicecontrol-instances/distributed-instances.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Multiple ServiceControl Instances
+reviewed: 2017-07-16
+component: ServiceControl
+---
+
+### Known Constraints / Issues
+
+- Only supports Audits
+- SC Master instances must be configured with Address and API URI of remotes - JSON string
+- Paging for message view in SI is wonky
+- Data from remote instances that cannot be reached by the master instance will not be included in the results
```

Comment:
What results? This is understandable to us who worked on the feature, but not a general userr.